agent/backend/pktvisor/pktvisor.go

```diff
@@ -269,93 +272,94 @@ func (p *pktvisorBackend) Start() error {
 	p.scraper = gocron.NewScheduler(time.UTC)
 	p.scraper.StartAsync()
 
-	//TODO create a startup for otel receiver / exporter component
-	ctx := context.Background()
-	p.exporter, err = createExporter(ctx, p.logger)
-	if err != nil {
-		p.logger.Error("failed to create a exporter", zap.Error(err))
-	}
-	p.receiver, err = createReceiver(ctx, p.exporter, p.logger)
-	if err != nil {
-		p.logger.Error("failed to create a receiver", zap.Error(err))
-	}
-
-	err = p.exporter.Start(ctx, nil)
-	if err != nil {
-		p.logger.Error("otel exporter startup error", zap.Error(err))
-		os.Exit(1)
-	}
-
-	err = p.receiver.Start(ctx, nil)
-	if err != nil {
-		p.logger.Error("otel receiver startup error", zap.Error(err))
-		os.Exit(1)
-	}
-
-	// scrape all policy json output with one call every minute.
-	// TODO support policies with custom bucket times
-	job, err := p.scraper.Every(1).Minute().WaitForSchedule().Do(func() {
-		metrics, err := p.scrapeMetrics(1)
+	if p.scrapeOtel {
```

Comment:
this is large enough to be hard to read now, please split each fork in this logic block here (setting up otel and setting up the other scraper) out to its own pktvisorBackend method